src/opentelemetry.erl

```diff
@@ -244,27 +244,18 @@ link(_, _, _, _) ->
       Attributes :: attributes().
 event(Name, Attributes) when is_binary(Name),
                              is_list(Attributes) ->
-    #event{time=timestamp(),
-           name=Name,
-           attributes=Attributes};
+    event(erlang:system_time(nanosecond), Name, Attributes);
```

Comment:
Why use nanoseconds instead of native and change when needed?

Reply:
I did originally but then realized I think it is more complicated that way. Since events represent something that could have happened outside the current process or even node and the user having a timestamp given to them from elsewhere it made more sense to just say "it must be in nano seconds" then to convert to native. So they still potentially have to convert to nano but still feels more right for some reason.

I guess it also keeps from a user potentially having an event happen on another node, return a timestamp to the process that wants to record it on its span in native and that being useless since it was from another node.

Reply:
Ok